java/src/main/java/com/adhoc/flight/utils/QueryUtils.java

```diff
@@ -0,0 +1,241 @@
+             *         VectorSchemaRoot batchRead = reader.getVectorSchemaRoot;
+             *         
+             *         // Updating the data inside the VectorSchemaRoot.
+             *         reader.loadNextBatch();
+             *         
+             *         // Printing out the results.
+             *         System.out.println(batchRead.contentToTSVString());
+             *     }
+             * }
+             * 
+             * ============================ INFO =============================
+             * 
+             * For more information on this, please refer to the documentation:
+             *     <https://arrow.apache.org/docs/java/ipc.html>
+             */
+            System.out.println(vectorSchemaRoot.contentToTSVString());
+
+            print(Fillers.FOOTER, "Number of records retrieved: " + vectorSchemaRoot.getRowCount());
+        }
+
+        /**
+         * Prints the provided exception to the console.
+         * 
+         * @param e the exception to be printed out.
+         */
+        public static void printExceptionOnClosed(Exception e) {
+            print(Prefixes.ERROR, e.getMessage());
+            e.printStackTrace();
+        }
+
+        private static void print(Prefixes prefixes, String message) {
+            System.out.println(prefixes.toFormattedString() + " " + message);
+        }
+        
+        private static void print(Fillers fillers, String message) {
+        	String filler = fillers.toFormattedString();
+        	System.out.println(filler + " " + message + " " + filler);
+        }
+    }
+}
+
+enum Fillers {
+    HEADER('-'),
+    FOOTER('=');
+
+    private static final int REPEAT = 18;
+    private final char character;
+
+    private String cache;
+    
+    Fillers(char character) {
+        this.character = character;
+    }
+
+	/**
+     * Reads the {@link String} from {@link #cache} or, if absent,
+     * creates a new {@code String} representation of this instance.
+     * 
+     * @return the {@code String} representation of this instance, formatted.
+     */
+    public final String toFormattedString() {
+        if (cache != null)
+            return cache;
+        
+        formatToString();
+        return toFormattedString();
+    }
+    
+    private void formatToString() {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        for (int current = 0; current < getRepeat(); current++) {
+            stringBuilder.append(character);
+        }
+
+        cache = stringBuilder.toString();
+    }
+    
+    private static final int getRepeat() {
+        return REPEAT;
+    }
+}
+
+enum Prefixes {
+    ERROR("ERROR"),
+    INFORMATION("INFO");
+
+	private static final char[] BRACKETS = new char[] {'[', ']'};
+	private final String string;
+	private String cache;
+
+	Prefixes(String string) {
+		Preconditions.checkArgument(string.length() > 0);
+	    this.string = string;
+	}
+
+	private static final char getOpenBrackets() {
```

Comment:
These methods just return left/right curly braces. I'd just use these characters directly, rather than get them from a method that returns a constant.